src/Plugin.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         DebugKit 3.15.0
+ * @license       https://www.opensource.org/licenses/mit-license.php MIT License
```

Comment:
This whole docblock needs to be removed. This file is not under CSF's copyright :).

Reply:
Thanks, it's done. Had a feeling it should have been removed.